### ISSUE TITLE
Primordial Leaves causing ticking exception on server player Eldritch_End-FABRIC-MC1.20.1-0.2.31

### ISSUE THREAD

**Issue by gamerpotion** [nvb-uy/Eldritch_End#19]
Hi, whenever players use an axe on primordial leaves,  Neruina throws them out with a ticking exception error. This is on Fabric 1.20.1 0.15.7 (Eldritch_End-FABRIC-MC1.20.1-0.2.31.jar).  There is no crashlog for them, just below is what is spotted in the server console when they are chucked from server.

```[02.04 20:52:09] [Server] [Server thread/WARN]: Neruina Caught Exception
[02.04 20:52:09] [Server] java.lang.ArrayIndexOutOfBoundsExceptionIndex -1 out of bounds for length 0
[02.04 20:52:09] [Server] 	at net.minecraft.class_182.method_799(class_182.java:45) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_182.test(class_182.java:20) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at java.util.function.Predicate.lambda$and$0(Predicate.java:69) ~[?:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_79.method_414(class_79.java:34) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_85.expand(class_85.java:67) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_64.method_16777(class_64.java:22) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_69.expand(class_69.java:44) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_55.method_345(class_55.java:58) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_55.method_341(class_55.java:96) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_52.method_328(class_52.java:86) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_52.method_320(class_52.java:103) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_52.method_319(class_52.java:116) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_52.method_51878(class_52.java:111) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_4970.method_9560(class_4970.java:274) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_4970$class_4971.method_26189(class_4970.java:1129) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_2248.method_9609(class_2248.java:333) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_2248.method_9511(class_2248.java:352) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_2248.method_9556(class_2248.java:431) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3225.method_14266(class_3225.java:275) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3225.method_14264(class_3225.java:106) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3222.method_5773(class_3222.java:464) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3218.method_18762(class_3218.java:739) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_1937.mixinextras$bridge$accept$205(class_1937.java) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at com.bawnorton.neruina.handler.NeruinaTickHandler.safelyTickEntities$notTheCauseOfTickLag(NeruinaTickHandler.java:115) ~[neruina-1.3.0-fabric+1.18.2-1.20.1.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_1937.wrapOperation$edn000$neruina$catchTickingEntities(class_1937.java:12048) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_1937.method_18472(class_1937.java:480) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3218.method_31420(class_3218.java:385) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_5574.method_31791(class_5574.java:54) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3218.method_18765(class_3218.java:353) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:897) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.class_3176.method_3813(class_3176.java:283) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[server-intermediary.jar:?]
[02.04 20:52:09] [Server] 	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[02.04 20:52:09] [Disconnect] SnowTheArcticFox has disconnected, reasonYou have been kicked due to a ticking exception on your player. Please remove the culprit mod or contact an admin.
[02.04 20:52:09] [Server] [Server thread/INFO]: SnowTheArcticFox left the game
[02.04 20:52:15] [Server] [JDA MainWS-ReadThread/INFO]: @snowythearcticfox#0000 > ok got the tree

**Comment by nvb-uy** [comment]
Will be fixed on 0.3.0

**Comment by nvb-uy** [comment]
Closing because duplicate of https://github.com/nvb-uy/Eldritch_End/issues/12
